### DISCUSSION TITLE
Multi Cluster Service Mirroring / removing original

### DISCUSSION THREAD

**Discussion by cham-d** [linkerd/linkerd2#7458] [Q&A] [answered]
Hello, i am new in linkerd, my question is:
 I have two clusters and the first cluster has two services (one named  server and one named client), the second cluster has only one service (server). Can i mirror the service "server" from the second cluster to the first and remove completely the service "server" in the first cluster?
I tried with traffic split (100 weight in the service of the second cluster), it works, but when i delete the service server of the first cluster it stops sending data to the second cluster.
Thank you.
**Edit
If i delete  the service "server" in the first cluster, before i create the traffic split, it doesn't work. If i first create the traffic split and after delete the service server in the first cluster, it works and sends the traffic to the second cluster.
Can i achieve this without using traffic split?

**Comment by kleimkuhler** [comment, marked as answer]
@cham-d I'm having a bit of a hard time following your situation; let me know based off my reply if I'm not following something.

In your 1 cluster, you have _three_ Services—2 normal (client and server) and 1 mirrored (server from cluster 2). You are deleting the server Service in cluster 1, and want the client to send to the mirrored server from cluster 2?

If that's the case, you should be able to target it directly by the mirrored service name (it will have the cluster named appended to it) or creating the TrafficSplit with a 100 weight to—again—the mirrored service name.

**Reply by cham-d** [reply]
Yep, that's exactly my situation! So, i need to change the target name from e.g. "server" to "server-east" or if i don't want to change the name,the solution is to create a traffic split with a 100 weight to the mirrored service?
If so, i got confused, because i thought, that i don't have to change the target name and the client is able automatically talk to the mirrored service instead of the old deleted one.
I just tried both and both of them working! Is there any difference between the two implementations?

**Reply by kleimkuhler** [reply]
Ah great to hear! The only difference is if you keep the TrafficSplit and for some reason you re-create the `server` Service on the first cluster, it will still send traffic to the second cluster which could be surprising.

If it's easy for your client to change the service it sends to—i.e. the one with the cluster name appended—I'd probably go with that. But there will not be any performance overhead with going through the TrafficSplit.

**Reply by cham-d** [reply]
Ok Thank you!
